src/main/java/com/sfuentes/census/ResultCensus.java

```diff
@@ -0,0 +1,19 @@
+package com.sfuentes.census;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RequiredArgsConstructor
+@Getter
+public class ResultCensus {
+  private final List<Integer> censusNumberCount = new ArrayList<>();
+
+  public void addCensusNumberSingleWomenBetweenSixteenAndTwentyOne(Card card) {
```

Comment:
I think this method has a very specific name but it actually accepts any Card for any gender and any age.
perhaps a better name is filterOnlySingleFemaleAgeFrom16To21
just a suggestion because Add seems too specific

Reply:
I get it, fixed